EKF/EKFGSF_yaw.cpp

```diff
@@ -79,14 +79,23 @@ void EKFGSF_yaw::update(const imuSample& imu_sample,
 			if (!bad_update) {
 				float total_weight = 0.0f;
 				// calculate weighting for each model assuming a normal distribution
+				const float min_weight = 1E-5f;
```

Comment:
```suggestion
				const float min_weight = 1e-5f;
```

Reply:
Changed in multiple places for consistency